app/src/main/java/ca/bc/gov/bchealth/ui/custom/PHNExtension.kt

```diff
@@ -0,0 +1,38 @@
+package ca.bc.gov.bchealth.ui.custom
+
+import androidx.core.widget.doOnTextChanged
+import androidx.fragment.app.Fragment
+import ca.bc.gov.bchealth.R
+import com.google.android.material.textfield.TextInputLayout
+
+/*
+* Created by amit_metri on 13,January,2022
+*/
+
+fun Fragment.validatePhnNumber(tipPhn: TextInputLayout, errorMessage: String): Boolean {
```

Comment:
`errorMessage` is not used here

Reply:
Also, this is not full PHN validation. Validation algorithm is here: https://github.com/FreshworksStudio/phn-validator/blob/master/src/index.ts

You might be able to do this in one line using zip, map, and reduce.

Reply:
I will look in to this and will apply it.